shellstub.py

```
@@ -0,0 +1,90 @@
+
+    def register(
+        self,
+        name: str,
+        *,
+        variants: list[dict] | None = None,
+        stdout: str = "",
+        stderr: str = "",
+        exit_code: int = 0,
+    ) -> None:
+        if variants is None:
+            variants = [dict(match=None, stdout=stdout, stderr=stderr, exit_code=exit_code)]
```

Ruff (0.11.9)

Unnecessary `dict()` call (rewrite as a literal)

Rewrite as a literal

(C408)